### ISSUE TITLE
[Enhancement] Button to swap to Pokémon Go

### ISSUE THREAD

**Issue by GiantForkLift** [omkarmoghe/Pokemap#274]
Would it be possible to include a button in app that will open up/switch to the Pokémon Go app?

**Comment by GiantForkLift** [comment]
Also, I do realize that is already an option with the built-in feature on most phones. however if this was possible in app it could help cut down on time switching between these two apps.

**Comment by omkarmoghe** [comment]
yeah for sure, should be possible with intents if we can get the package name http://stackoverflow.com/a/7574735

might not make it into this friday's release (we're feature freezing to fix all the bugs), but def something we could look into

**Comment by s7092910** [comment]
@omkarmoghe Already got the package name. Also wouldn't be hard to get it either.

**Comment by GiantForkLift** [comment]
Awesome! Im thoroughly impressed with how quick you are all to respond and provide updates. If only everyone was like you.

And I am not trying to be picky or obtrusive, it's your project, but what if the icon where you push to find your location was actually an in app menu where you press/hold the icon down and three more icons appear that are for "Find my location", "Open Pokémon Go", and "Settings". It would eliminate the top bar and create more space for the map. I don't know if it's possible or not but it could have potential! I can create a new issue to better address this if you want.

**Comment by s7092910** [comment]
We will probably keep the Toolbar. But we might change where certain things are
